packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 52.7.0 - July 30, 2021
+
+## Added
+
+#### salesforcedx-vscode-apex
+
+- Introduce 3 new Apex Test Suite related commands to the command palette ([PR #3435](https://github.com/forcedotcom/salesforcedx-vscode/pull/3435))
+
+#### salesforcedx-vscode-core
+
+- Add command to logout of default org ([PR #3428](https://github.com/forcedotcom/salesforcedx-vscode/pull/3428))
```

Comment:
@AnanyaJha and @sbudhirajadoc - Should we add the actual command name for the logout to be consistent with the changelog entry for the Apex commands we added?  I feel like we might as well include the command name.  Thoughts?

Reply:
```suggestion
- Add command `SFDX: Log Out from Default Org` ([PR #3428](https://github.com/forcedotcom/salesforcedx-vscode/pull/3428))
```